docs/cendr.md

```diff
@@ -2,275 +2,394 @@
+1. Add new wild isolate sequence data, and process with the [trim-fq-nf](pipeline-trimming.md) pipeline.
+1. Align reads with [alignment-nf](pipeline-alignment.md).
+1. Call variants with [wi-gatk](pipeline-wiGATK.md).
+1. Identify new isotypes using the [concordance-nf](pipeline-concordance.md).
+1. Update the *C. elegans* WI Strain Info [spreadsheet](https://docs.google.com/spreadsheets/u/1/d/10x-CcKNCl80F9hMcrGWC4fhP_cbekSzi5_IYBY2UqCc/edit?usp=drive_web&ouid=113810443554564955854) with the new isotypes and update the release column to reflect the release date.
+1. Perform population genetic analysis with [post-gatk-nf](pipeline-postGATK.md).
+1. [Impute](pipeline-impute.md) the VCF.
+1. Annotate the VCF with the [annotation-nf](pipeline-annotation-nf.md) pipeline.
 
-```
+Pushing a new release requires a series of steps described below.
 
-If you do not see the full set of commands there - something is broken.
+### Uploading BAMs to Google Storage
 
-### Setup Credentials
+You will need Google Cloud credentials to upload BAMs to Google Storage.
 
-1. Authenticate with gcloud.
-2. Run the following command:
+Install the [Google Cloud SDK](https://cloud.google.com/sdk/docs/install) and configure with your GCP credentials and the `caendr` project ID.
 
 ```bash
-mkdir -p env_config
-flask decrypt_credentials
+gcloud init 
 ```
 
-This will create a directory with the site credentials (`env_config`). Keep these secret.
-
-!!! important
-    __DO NOT COMMIT THESE CREDENTIALS TO GITHUB__!!!
-
-### Load the database
+Once configured, navigate to the BAM location on b1059.
 
-The site uses an SQLite database that can be setup by running:
+```bash
+# CD to bams folder...
+cd /projects/b1059/data/c_elegans/WI/alignments/
+```
+Run this command in screen to ensure that it completes (it's going to take a while)
 
 ```
-flask download_db
+gsutil rsync . gs://caendr-site-private-bucket/bam/c_elegans/
 ```
 
-This will update the SQLite database used by CeNDR (`base/cendr.db`). The tables are:
-
-* `homologs` - A table of homologs+orthologs.
-* `strain` - Strain info pulled from the google spreadsheet __C. elegans WI Strain Info__.
-* `wormbase_gene` - Summarizes gene information; Broken into component parts (e.g. exons, introns etc.).
-* `wormbase_gene_summary` - Summarizes gene information. One line per gene.
-* `metadata` - tracks how data was obtained. When. Where. etc.
-
-### Test the site
+### Adding isotype images
 
-You can at this point test the site locally by running:
+Isolation photos are initially prepared on dropbox and are located in the folder here:
 
 ```bash
-flask run
+~/Dropbox/Andersenlab/Reagents/WormReagents/isolation_photos/c_elegans
 ```
 
-Be sure you have direnv. Otherwise you should source the .envrc file prior to running:
+Each file should be named using the isotype name and the strain name strain name in the following format:
 
 ```bash
-source .envrc
-flask run
+<strain>.jpg
```

Comment:
Looks like on dropbox these are stored as isotype_strain.jpg, but on gcp just strain.jpg? Just want to verify we only need strain name here.